src/interface/icons/Uptime.js

```diff
@@ -3,7 +3,12 @@ import React from 'react';
 // https://thenounproject.com/term/duration/370713/
 // duration by Bohdan Burmich from the Noun Project
 const Icon = ({ ...other }) => (
-  <svg xmlns="http://www.w3.org/2000/svg" viewBox="10 10 80 80" className="icon" {...other}>
+  <svg
```

Comment:
im assuming Prettier automatically formatted this. No real reason to change this from how it was before.

Reply:
I would argue that any code style changes that do not make the linter fail should be accepted.
But I can change it back as you ask.